pages/congrats/index.tsx

```diff
@@ -9,8 +9,10 @@ import ConfirmationGraphic from "../../src/components/ConfirmationGraphic";
 import useFormDataContext from "../../src/context/FormDataContext";
 import { FormResults } from "../../src/interfaces";
 import { homePageRedirect } from "../../src/utils/utils";
+import { cookieDomain } from "../../appConfig";
 
 function ConfirmationPage(): JSX.Element {
+  console.log("render confirmation");
```

Comment:
Hmm I think this is actually ok to keep for prod debugging, but can you update the message so it's clearer in the browser and logs when read.

Reply:
oh it should be removed. it was just for testing.